src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Bulldozer(),new Excavator(),new Truck()};
```

Comment:
Consider using abstract references instead of specific ones for the `machines` array. This allows you to take advantage of polymorphism and makes the code more flexible. For example:
`Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};`